src/Sulu/Bundle/SearchBundle/Resources/public/scss/modules/dropdown-input.scss

```diff
@@ -44,6 +44,11 @@
         }
     }
 
+    .husky-select-label {
+        height: 40px;
+        line-height: 40px;
+    }
```

Comment:
The html structure of the select changed in the linked husky-PR. This made the change you commented on necessary.